docs/Config_Reference.md

```diff
@@ -4659,6 +4659,15 @@ scale.
 [load_cell]
 sensor_type:
 #   This must be one of the supported sensor types, see below.
+#counts_per_gram:
+#   The floating point number of sensor counts that indicates 1 gram of force.
+#   This value is calculated by the CALIBRATE_LOAD_CELL command.
+#reference_tare_counts:
+#   The integer tare value, in raw sensor counts, taken when CALIBRATE_LOAD_CELL
+#   is run. This is the default tare value when klipper starts up.
+#reverse:
+#   Reverses the polarity of the load cell. This is a boolean value, the
+#   default is False.
```

Comment:
Out of curiosity, is "reverse" effectively the same as a negative "counts_per_gram"?